src/ert/scheduler/scheduler.py

```diff
@@ -102,7 +102,10 @@ def __init__(
 
     def kill_all_jobs(self) -> None:
         assert self._loop
-        asyncio.run_coroutine_threadsafe(self.cancel_all_jobs(), self._loop)
+        # Checking that the loop is running is required because everest is closing the
+        # simulation context every an optimization simulation batch is done
```

Comment:
I understand the comment, but did you mean `whenever` instead of `every` ? This is nitpick but the there is something not quite right with this.

Reply:
I wanted to write `every time` but I think `whenever` is better :)